circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  cache_directories:
+    - "~/nilearn_data"
+  override:
+    - pip install --upgrade pip
+    - pip install -e .
+    - pip install sphinx matplotlib coverage Pillow
+
+test:
+  override:
+    - make
```

Comment:
Also it seems like `make all` in nilearn is `make clean test doc-noplot` do you probably want to use `make clean test` instead.